dist-material/release-docs/LICENSE

```diff
@@ -340,6 +340,7 @@ The text of each license is the standard Apache 2.0 license.
     https://mvnrepository.com/artifact/io.vavr/vavr/0.10.3 Apache-2.0
     https://mvnrepository.com/artifact/io.vavr/vavr-match/0.10.3 Apache-2.0
     https://mvnrepository.com/artifact/io.zipkin.zipkin2/zipkin/2.23.16 Apache-2.0
+    https://mvnrepository.com/artifact/io.zipkin/zipkin-lens/2.23.16 Apache-2.0
```

Comment:
I think we should resolve lens UI's dependencies, and regenerate the LICENSE file

Reply:
To regenerate the license file (and make it possible to automatically generate it as what we are doing now), we have to copy the `package.json` and `package-lock.json` from Zipkin Lens project, also Zipkin Lens put all dependencies (including development dependencies) into `dependencies` section so we would also include many un-distributed licenses.

Reply:
OK, this seems a little crazy :(
It looks like we are over declaring more than we should be.

Reply:
Let's keep the lib dependencies for now. Copy the package and lock files seem easy to be out-of-date too.

Reply:
> Let's keep the lib dependencies for now. Copy the package and lock files seem easy to be out-of-date too.

I don't think we will update the lens dependency too often, this feature is not in our main path and the lens dependency has only static compiled html/css/js files